alvr/dashboard/src/dashboard/components/notifications.rs

```diff
@@ -18,7 +18,7 @@ const TIMEOUT: Duration = Duration::from_secs(5);
 const NO_NOTIFICATIONS_MESSAGE: &str = "No new notifications";
 const NOTIFICATION_TIPS: &[&str] = &[
     // The following tips are ordered roughtly in the order settings appear
-    r#"If you started having crashes after changing some settings, reset ALVR by deleting "session.json"."#,
+    r#"If you started having crashes after changing some settings, reset ALVR by re-running "Run setup wizard" from "Installation" tab and resetting settings in wizard, or deleting "session.json" from ALVR folder."#,
```

Comment:
I feel like we should delete the session.json part alltogether because I think the folder where that's stored is just in random places depending on the os.